mailchimp_ecommerce.module

```diff
@@ -699,6 +699,40 @@ function mailchimp_ecommerce_get_order($order_id) {
   return NULL;
 }
 
+/**
+ * Gets a cart from the current MailChimp store.
+ *
+ * @param string $cart_id
+ *   The cart ID.
+ *
+ * @return object
```

Comment:
The return is mixed. An object or NULL can be returned.

Reply:
Good call.